src/main/java/uk/gov/pay/directdebit/payments/api/PaymentViewValidator.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.pay.directdebit.payments.api;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static com.google.common.collect.Lists.newArrayList;
+
+public class PaymentViewValidator {
+
+    private static final Long MAX_PAGE_NUMBER = 500l;
+    private static final Long DEFAULT_PAGE_NUMBER = 1l;
+
+    public static Pair<Long, Long> validatePagination(Optional<Pair<Long, Long>> paginationParams) {
```

Comment:
Also, i'd rather have a typed `PaginationParams` class, with two long fields (`maxPageNumber` and `defaultPageNumber`) rather than`Pair<Long, Long>`.

`paginationParams.getMaxPageNumber()` is really so much more clear than `paginationParams.getLeft()`